### PR TITLE
20634: MINOR Added Notebook to demonstrate alpha FeaturesAttributesControl widget

### PR DESCRIPTION
**Pull request by lancegliser**

### CODE REVIEW

File: recipes/1-Insights/engine-insights.ipynb
Comment (bbefort) [inline recipes/1-Insights/engine-insights.ipynb:1]: Did you also add the widget to this recipe? Or was this to remove it?

File: recipes/5-Technical_Validation/alpha_infer_feature_attributes_tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,6451 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c98fa8fe",
+   "metadata": {},
+   "source": [
+    "For interactive configuration, you may wish to launch the included FeaturesAttributesControl widget.\n",
+    "\n",
+    "This functionality is currently in `alpha` status. You may safely rely on the params output from produced, but the python to load, render, and pull values out of the widget are expected to change."
```
Comment (bbefort) [inline recipes/5-Technical_Validation/alpha_infer_feature_attributes_tutorial.ipynb:220]: produced what? this sentence is a bit confusing

**Review by bbefort — COMMENTED** [review]
Add a period after "get a highlight of the features for read only view." in section 2.

File: recipes/5-Technical_Validation/alpha_infer_feature_attributes_tutorial.ipynb
Comment (bbefort) [inline recipes/5-Technical_Validation/alpha_infer_feature_attributes_tutorial.ipynb:1]: Can you make the sections (all with the same number of markdown hashes):
1) Manual inference
2) Interactive inference
3) Caveats

Comment (bbefort) [reply recipes/5-Technical_Validation/alpha_infer_feature_attributes_tutorial.ipynb:1]: For each section, you can have an "infer attributes" subsection and a "review attributes" subsection

Comment (bbefort) [inline recipes/5-Technical_Validation/alpha_infer_feature_attributes_tutorial.ipynb:1]: "With your Trainee configured, you are now free to use it for further _Howso_ operations"

Comment (bbefort) [inline recipes/5-Technical_Validation/alpha_infer_feature_attributes_tutorial.ipynb:1]: Under caveats, I think you need to break up the sentence "Some providers..." into two parts: add a period after "a single bundle." and then start a new sentence with "You may use..."

**Review by bbefort — CHANGES_REQUESTED** [review]

**Review by bbefort — DISMISSED** [review]

**Comment by lancegliser** [comment]
[1:59 PM] Jack Xia
> You mind if we hold off on those recipe merges for a day or two? Unless you urgently need them soon?

**Review by bbefort — APPROVED** [review]

**Pull request merged**